CHANGELOG

```diff
@@ -1,3 +1,14 @@
+snowplow-web 0.14.1 (2023-05-23)
```

Comment:
snowplow-web 0.14.1 shouldn't this be aligned to the fix version?

Reply:
I had to name the branch something different as we already have the 0.14.1 branch (which will end up being 0.15.0), but the version of the package is 0.14.1

Reply:
just wanted to remove the comment but you are right